.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an improvement
+title: "[Feature]"
```

Comment:
Do you think it might help to have a kind of dummy description aka *title of feature* telling the user to actually extend the `title`? If yes, this also applies for the bug template.

Reply:
How about this? I guess we do not need ``[Feature]`` tag if we have default labels.
```suggestion
title: "<Enter short description of desired feature>"
```